libminifi/src/provenance/Provenance.cpp

```diff
@@ -178,8 +181,8 @@ bool ProvenanceEventRecord::Serialize(org::apache::nifi::minifi::io::BufferStrea
 
   if (this->_eventType == ProvenanceEventRecord::FORK || this->_eventType == ProvenanceEventRecord::CLONE || this->_eventType == ProvenanceEventRecord::JOIN) {
     // write UUIDs
-    uint32_t number = this->_parentUuids.size();
-    ret = outStream.write(number);
+    uint32_t parent_uuids_size = gsl::narrow<uint32_t>(this->_parentUuids.size());
```

Comment:
an alternative name could be `parent_uuids_count`, although not sure if it is "better"

Reply:
yes, I think it's slightly better -- changed it to `..._count`